Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -7873,7 +7872,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
-				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```

Comment:
Why did all the stuff in this file get reverted?

Reply:
My Xcode is doing this : Version 10.2 (10E125)